packages/dashboard/src/customization/widgets/lineSymbol/lineAnchor/component.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { CSSProperties, useEffect, useRef } from 'react';
+import './component.css';
+import { LineWidget } from '~/customization/widgets/types';
+import { XYCoord, useDrag } from 'react-dnd';
+import { DashboardWidget } from '~/types';
+import { useGridSettings } from '~/components/actions/useGridSettings';
+import { useWidgetActions } from '~/customization/hooks/useWidgetActions';
+
+function computeNewPosition(
```

Comment:
we should add jsdoc comments to these functions. Also any reason to mix regular / arrow functions?

Reply:
added jsdoc comments to all functions (including the hook) in this file. no reason in particular, happened with the flow - made them arrow functions.